templates/board.md

```diff
@@ -0,0 +1,23 @@
+---
+breaks:  false # See: https://github.com/hackmdio/codimd/issues/40#issuecomment-172927690
+tags: Hypha
+---
+# 2021-MM-DD Hypha Worker Co-op Board Meeting - Mmmm DD, 202X
+
+Time:       5:00-6:00pm ET  
+Location:   https://link.hypha.coop/calls  
+Attending:  ...  
+Cultivator: ...  
+Notetaker:  ... 
```

Comment:
Thoughts on adding a choice emoji to front of each line here? Def no block, but I do love like these little guys, and happy to use quirky Hypha-style ones :) Also, happy to spawn a new ticket and not clog this one! Feels like an opp for a little bit of personality when ppl cruise our notes